### PR TITLE
Api endpoints to display and edit alignments

### PR DESCRIPTION
**Pull request by bijob89**
## Description of the Change
Added API endpoints to check available books, chapter and verse information.
Added API endpoint to display alignments (input ->bcv)
Added API endpoint to edit a whole verse alignment(input -> bcv, posiitonal pairs)
Added dependencies information to the requirements.txt file for future installs.
Removed Hard-coded urls. Urls will be fetched as environment variables.

### CODE REVIEW

File: mt2414/main.py [hunk 1]
```diff
@@ -56,6 +70,21 @@ def get_db():
         g.db = psycopg2.connect(dbname=postgres_database, user=postgres_user, password=postgres_password, host=postgres_host, port=postgres_port)
     return g.db
 
+def digit_length_check(num):
```
Comment (beniza) [inline mt2414/main.py]: you can use _str_.**zfill()** to achieve the same result.
```python
num = [1, 11, 111]
for i in num:
    print(str(i).zfill(3))
```

-----output----
001
011
111

File: mt2414/main.py [hunk 2]
```diff
@@ -98,9 +127,9 @@ def new_registration():
     body = '''Hi,<br/><br/>Thanks for your interest to use the AutographaMT web service. <br/>
     You need to confirm your email by opening this link:
 
-    <a href="https://api.mt2414.in/v1/verifications/%s">https://api.mt2414.in/v1/verifications/%s</a>
+    <a href="https://%s/v1/verifications/%s">https://%s/v1/verifications/%s</a>
 
-    <br/><br/>The documentation for accessing the API is available at <a href="http://docs.mt2414.in">docs.mt2414.in</a>''' % (verification_code, verification_code)
+    <br/><br/>The documentation for accessing the API is available at <a href="https://docs.autographamt.com">https://docs.autographamt.com</a>''' % (host_api_url, verification_code, host_api_url, verification_code)
```
Comment (beniza) [inline mt2414/main.py]: +1

File: mt2414/main.py (hunk 1 above)
Comment (beniza) [reply mt2414/main.py]: You could optionally use lambda operator
```python
>>> num = [1, 11, 111]
>>> filled_num = map(lambda x: str(x).zfill(3), num)
>>> list(filled_num)
['001', '011', '111']
```

File: mt2414/main.py [hunk 3]
```diff
@@ -1217,3 +1246,152 @@ def corrections():
 @check_token
 def suggestions():
     return '{}\n'
+
+@app.route('/v2/alignments/view', methods=["GET", "POST"])
```
Comment (beniza) [inline mt2414/main.py]: API endpoints are nouns, not verbs

Comment (beniza) [reply mt2414/main.py]: Could you add the `docstrings` for all the APIs and methods?

File: mt2414/main.py [hunk 4]
```diff
@@ -1217,3 +1242,176 @@ def corrections():
 @check_token
 def suggestions():
     return '{}\n'
+
+@app.route('/v2/alignments/<bcv>', methods=["GET"])
+def getalignments(bcv):
+    '''
+    Returns list of positional pairs, list of Hindi words, list of strong numbers for the bcv queried.
```
Comment (beniza) [inline mt2414/main.py]: list of `Hindi` words > list of `Target Language` words

File: mt2414/main.py (hunk 1 above)
Comment (bijob89) [reply mt2414/main.py]: This has been changed

File: mt2414/main.py (hunk 3 above)
Comment (bijob89) [reply mt2414/main.py]: Both the above have been done and submitted.

**Pull request merged**